package.json

```
@@ -100,7 +100,7 @@
     ]
   },
   "engines": {
-    "node": ">=10.16.0 <=12.x"
+    "node": ">=10.16.0 <=16.x"
```

```suggestion
    "node": ">=12.x <16.x",
    "yarn": ">=1.19.2 <2.x"
```

There is no `16` yet and in fact `< 15.x` should be more accurate since the current active [LTS](https://nodejs.org/en/about/releases/#releases) releases are `12.x` and `14.x`